pkg/server/server_authz_test.go

```diff
@@ -0,0 +1,1570 @@
+package server
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/oklog/ulid/v2"
+	openfgav1 "github.com/openfga/api/proto/openfga/v1"
+	language "github.com/openfga/language/pkg/go/transformer"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/goleak"
+	"google.golang.org/protobuf/testing/protocmp"
+	"google.golang.org/protobuf/types/known/wrapperspb"
+
+	"github.com/openfga/openfga/pkg/authclaims"
+	"github.com/openfga/openfga/pkg/authz"
+	"github.com/openfga/openfga/pkg/storage/memory"
+	"github.com/openfga/openfga/pkg/tuple"
+	"github.com/openfga/openfga/pkg/typesystem"
+)
+
+type store struct {
+	id      string
+	modelID string
+}
+
+type authzSettings struct {
+	openfga  *Server
+	clientID string
+	root     *store
+	test     *store
+}
+
+const (
+	rootStoreModel = `
```

Comment:
I see the notion of a `rootStoreModel` here but it appears to only exist in a testing context. Where is the actual model?

Reply:
The idea is that users can create their own model that uses our expected relations. This model can be used as an example in documentation.